packages/@uppy/core/src/index.js

```diff
@@ -428,7 +414,46 @@ class Uppy {
    */
   getFiles () {
     const { files } = this.getState()
-    return Object.keys(files).map((fileID) => files[fileID])
+    return Object.values(files)
+  }
+
+  getObjectOfFilesPerState () {
+    const { files: filesObject, totalProgress, error } = this.getState()
+    const files = Object.values(filesObject)
+    const inProgressFiles = files.filter(
+      (file) => !file.progress.uploadComplete && file.progress.uploadStarted
+    )
+    const newFiles =  files.filter((file) => !file.progress.uploadStarted)
+    const startedFiles = files.filter(
+      file => file.progress.uploadStarted || file.progress.preprocess || file.progress.postprocess
+    )
```

Comment:
nit: to leave it as one-liner
```suggestion
    const startedFiles = files.filter(({ progress }) => progress.uploadStarted || progress.preprocess || progress.postprocess)
```

Reply:
This will exceed our line length limit.

Reply:
My suggestion is 123 char long, isn't the limit fixed at 125?